src/enochecker/enochecker.py

```diff
@@ -359,37 +218,43 @@ def _setup_logger(self) -> None:
         self.critical: Callable[..., None] = self.logger.critical
 
     @property
-    def current_round(self) -> Optional[int]:
+    def ctx(self) -> str:
         """
-        Deprecated! Only for backwards compatibility! Use self.round instead.
+        short hand version for self.task_chain_id
 
-        :return: current round
+        :return: self.task_chain_id
         """
-        warnings.warn(
-            "current_round is deprecated, use round instead", DeprecationWarning
-        )
-        return self.round
+        return self.task_chain_id
 
     @property
-    def noise(self) -> Optional[str]:
+    def noise(self) -> str:
         """
-        Pretty similar to a flag, just in a different mode (storeNoise vs storeFlag).
+        Creates a stable noise value for the current context.
```

Comment:
like the other comments s/context/chain/

Reply:
Chain context?